### DISCUSSION TITLE
Attachement Mode

### DISCUSSION THREAD

**Discussion by brunnorpdias** [darlal/obsidian-switcher-plus#142] [Ideas]
Hello. Absolutely love this plugin, thank you! I just wanted to float the idea of a attachment mode. I prefer for the switcher to normally just show me MD files as it makes finding pages much easier. But, recently, I thought that a mode where I can specifically search for attachments (non MD files) would be super useful, and as far as I know there is no way to currently do this... Does anyone have this need as well?

**Comment by darlal** [comment]
Glad you're finding the plugin useful! Thanks for the suggestion, I think we could pretty easily enable filtering for non-markdown files without having to create a whole new separate mode. For example a [Quick Filter](https://github.com/darlal/obsidian-switcher-plus#quick-filters) could be added that enables you to easily restrict the search to only non-markdown files using only a single hotkey. Something like that could work pretty well in this case.

**Comment by brunnorpdias** [comment]
Thanks for your response. The problem is that Quick Filters only work for the active file and not the whole vault, right?

**Comment by darlal** [comment]
The Quick Filters available currently are not vault wide, but there's no inherent limitation. I'm planning to add several other vault wide Quick Filters and this would be a good one to add as well.

**Comment by darlal** [comment]
[Release 4.1.0](https://github.com/darlal/obsidian-switcher-plus/releases/tag/4.1.0) introduces the `external files` [Quick Filter](https://github.com/darlal/obsidian-switcher-plus?tab=readme-ov-file#quick-filters) to limit the results to non `.md`/`.canvas` file extensions.
